analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -94,13 +105,16 @@ class NuGetSupport {
         }
     }
 
-    private val serviceIndex = runBlocking { mapFromUrl<ServiceIndex>(JSON_MAPPER, SERVICE_INDEX_URL) }
+    private val serviceIndices = serviceIndexUrls.map {
+        runBlocking { mapFromUrl<ServiceIndex>(JSON_MAPPER, it) }
+    }
```

Comment:
For the fun of it, we should be able to map URLs concurrently using

    private val serviceIndices = runBlocking {
        serviceIndexUrls.map {
            async { mapFromUrl<ServiceIndex>(JSON_MAPPER, it) }
        }.awaitAll()
    }

Reply:
FYI, I find https://kotlinexpertise.com/kotlin-coroutines-concurrency/ and https://jivimberg.io/blog/2018/05/04/parallel-map-in-kotlin/ to be good reads.